src/components/Posts/PostComments/Comments.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import PostComments from "./PostComments";
+import PostTimeStamps from '../PostBody/PostTimeStamps';
+import '../post.css'
+
+
+class Comments extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+    userInput: '',
+    comments: this.props.obj.comments,
+    timestamps: this.props.obj,
+    counter: this.props.obj.likes
+    }
+}
+
+usersInput = (event) => {
+    const {value} = event.target;
+    this.setState({
+        userInput: value
+    });
+}
+
+addComment = (event) => {
+    event.preventDefault();
+    const newComment = {
+        username: "basicCoder45",
```

Comment:
Good work achieving comment functionality. Try storing username in local storage, and making it dynamic with a login screen.